samples/snippets/src/main/java/com/example/bigquerystorage/WritePendingStream.java

```diff
@@ -90,6 +90,10 @@ public static void writePendingStream(String projectId, String datasetName, Stri
               .build();
       BatchCommitWriteStreamsResponse commitResponse =
           client.batchCommitWriteStreams(commitRequest);
+      // If the response does not have a commit time, it means the commit operation failed.
+      if (commitResponse.hasCommitTime() == false) {
```

Comment:
Does it not return an error object in this case? User is expected to guess that something went wrong?

Reply:
The user can call getStreamErrorsList to get a list of StorageError's. You could use this to look at the error code per stream. Does it make sense to show this?

Reply:
I think it would make more sense to just show the error objects rather than show hasCommitTime indirectly.

Reply:
Per @yirutang checking for hasCommitTime is more semantically meaningful, because that's the error condition. I added code to display the list of error messages.

Reply:
Got it. SGTM.